webapp/tests/karma/ts/modules/contacts/contacts.component.spec.ts

```diff
@@ -277,7 +389,8 @@ describe('Contacts component', () => {
       component.ngOnInit();
       flush();
 
-      expect(contactTypesService.getChildren.args[0].length).to.equal(0);
+      expect(contactTypesService.getChildren.callCount).to.equal(2);
```

Comment:
How come `contactTypesService.getChildren` is called twice?

Reply:
two different process are calling the function on initialisation

Reply:
Which ones? are they both required?

Reply:
One is for defaultFilters looks like it's used for search, and the other one for action bars, different parameters. I moved the action bar to contact content component so this one is back to original numbers.